### PR TITLE
Minor corrections

### PR DESCRIPTION
**Pull request by gergo85**
- Remove deprecated path symbols
- Change the $implement value
- Rename the list_config.yaml file

### CODE REVIEW

**Comment by LukeTowers** [comment]
$ isn't deprecated and using ~ instead is incorrect. $ is for the plugins directory and ~ is for the application base path

**Comment by gergo85** [comment]
I see. I modified it.

**Comment by LukeTowers** [comment]
Theres still paths that have to be fixed.

**Comment by bennothommo** [comment]
@gergo85 I think you might have the `~` and `$` the wrong way around in the paths you've changed.

> $ is for the plugins directory and ~ is for the application base path

**Comment by gergo85** [comment]
I replaced all ~ simbol.

File: backend-lists.md [hunk 1]
```diff
@@ -371,7 +371,7 @@ To display a column that shows the number of related records, use the `useRelati
 
     content:
         type: partial
-        path: ~/plugins/acme/blog/models/comments/_content_column.htm
+        path: $/plugins/acme/blog/models/comments/_content_column.htm
```
Comment (LukeTowers) [inline backend-lists.md:374]: Now you've broken it again. $/plugins = ~/plugins/plugins

**Pull request closed without merging**